momento-sdk/src/main/java/momento/sdk/ValidationUtils.java

```diff
@@ -23,6 +23,7 @@ public final class ValidationUtils {
   static final String SCORE_RANGE_INVALID =
       "maxScore (inclusive) must be greater than or equal to minScore (inclusive).";
   static final String SIGNING_KEY_TTL_CANNOT_BE_NEGATIVE = "Signing key TTL cannot be negative.";
+  static final String TRUNCATE_TO_SIZE_CANNOT_BE_NEGATIVE = "Cannot truncate to a negative size";
```

Comment:
We do not want to allow zero either, so this should say "Cannot truncate to a non-positive size".